### PR TITLE
Add getting user from `chat_boost`

### PR DESCRIPTION
**Pull request by Desiders**
Close: #1472

### CODE REVIEW

**Comment by Olegt0rr** [comment]
@Desiders, please add some related tests

**Comment by Desiders** [comment]
> @Desiders, please add some related tests

I added this, but I'm not sure that this test is right in this case

File: tests/test_dispatcher/test_middlewares/test_user_context.py [hunk 1]
```diff
@@ -41,3 +42,46 @@ async def test_call(self):
...
+        event_context = data["event_context"]
+        assert isinstance(event_context, EventContext)
+        assert event_context.chat is chat
+        assert event_context.user is None
+        assert event_context.thread_id is None
+        assert data["event_chat"] is chat
+        assert data.get("event_from_user") is None
+        assert data.get("event_thread_id") is None
+
+        data.clear()
```
Comment (Olegt0rr) [inline tests/test_dispatcher/test_middlewares/test_user_context.py:87]: Use parametrize to repeat the test with other params

Comment (Olegt0rr) [reply tests/test_dispatcher/test_middlewares/test_user_context.py:87]: Also try to find other tests covering neighbour data in context. Looks like we just need to add params to existing test

Comment (Desiders) [reply tests/test_dispatcher/test_middlewares/test_user_context.py:87]: I don't understand what you mean by 
> Looks like we just need to add params to existing test

`test_call` test or the test that I wrote?
I don't see the point in the first test and how to cover the current case.

**Review by JrooTJunior — APPROVED** [review]

**Pull request merged**